src/tiled/abstractobjecttool.cpp

```diff
@@ -160,7 +161,10 @@ void AbstractObjectTool::mouseMoved(const QPointF &pos,
     const QPointF tilePosF = mapDocument()->renderer()->screenToTileCoords(offsetPos);
     const int x = qFloor(tilePosF.x());
     const int y = qFloor(tilePosF.y());
-    setStatusInfo(QString(QLatin1String("%1, %2 (%3, %4)")).arg(x).arg(y).arg(pixelPos.x()).arg(pixelPos.y()));
+    setStatusInfo(QString(QLatin1String("%1, %2 (%3, %4)")).arg(x)
+                                                           .arg(InvertYAxisHelper(mapDocument()).tileY(y))
+                                                           .arg(pixelPos.x())
+                                                           .arg(InvertYAxisHelper(MapDocumentActionHandler::instance()->mapDocument()).pixelY(pixelPos.y())));
```

Comment:
```suggestion
                                                           .arg(InvertYAxisHelper(mapDocument()).pixelY(pixelPos.y())));
```
You got it right for `tileY`, why not for `pixelY`? :-)